src/common/config_parser.cpp

```diff
@@ -372,7 +372,7 @@ void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
   cli.add<size_t>("--sentencepiece-max-lines",
       "Maximum lines to train SentencePiece vocabulary, selected with sampling from all data. "
```

Comment:
If there is no sampling anymore, please update "selected with sampling from all data" in the help message.

Reply:
I keep our sampling in there since it also concatenates the files from both translation directions. So there is more logic happening in the background.

Reply:
As for the changes, I will ask Aaron to review them internally and do a separate PR when syncing changes back. I think most of the concerned protobuf which is now gone.

Reply:
Leaving the description as is since we are still sampling.